x-pack/plugins/security_solution/public/overview/components/entity_analytics/common/basic_table_without_border_bottom.tsx

```diff
@@ -8,10 +8,30 @@ import styled from 'styled-components';
 import { EuiBasicTable } from '@elastic/eui';
 
 // @ts-expect-error TS2769
-export const BasicTableWithoutBorderBottom = styled(EuiBasicTable)`
+export const StyledBasicTable = styled(EuiBasicTable)`
```

Comment:
can you please rename this file as well?

Reply:
Good catch!